### ISSUE TITLE
"Internal Server Error" when trying to view 

### ISSUE THREAD

**Issue by redstreet** [aumayr/fava#348]
If I click on the date of a transaction to view its context, open it up in the editor, etc., I get:

![image](https://cloud.githubusercontent.com/assets/691755/17606860/9aa8a0d0-5fd6-11e6-832d-85b58ce52a24.png)

Example: click on any date in:
http://fava.pythonanywhere.com/example-beancount-fava-0-3-0-2016-08-11-pypi/account/Expenses:Financial:Commissions/

Eg:
http://fava.pythonanywhere.com/example-beancount-fava-0-3-0-2016-08-11-pypi/context/0b6775736babcf12638eef23f4993c6a/

````jinja2.exceptions.TemplateSyntaxError
jinja2.exceptions.TemplateSyntaxError: invalid syntax for function call expression

Traceback (most recent call last)
File "/usr/lib/python3.4/site-packages/flask/app.py", line 2000, in __call__
return self.wsgi_app(environ, start_response)
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1991, in wsgi_app
response = self.make_response(self.handle_exception(e))
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1567, in handle_exception
reraise(exc_type, exc_value, tb)
File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
raise value
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1988, in wsgi_app
response = self.full_dispatch_request()
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1641, in full_dispatch_request
rv = self.handle_user_exception(e)
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1544, in handle_user_exception
reraise(exc_type, exc_value, tb)
File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
raise value
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1639, in full_dispatch_request
rv = self.dispatch_request()
File "/usr/lib/python3.4/site-packages/flask/app.py", line 1625, in dispatch_request
return self.view_functions[rule.endpoint](**req.view_args)
File "/usr/lib/python3.4/site-packages/fava/application.py", line 164, in context
return render_template('context.html', ehash=ehash)
File "/usr/lib/python3.4/site-packages/flask/templating.py", line 133, in render_template
return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 851, in get_or_select_template
return self.get_template(template_name_or_list, parent, globals)
File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 812, in get_template
return self._load_template(name, self.make_globals(globals))
File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 786, in _load_template
template = self.loader.load(self, name, globals)
File "/usr/lib/python3.4/site-packages/jinja2/loaders.py", line 125, in load
code = environment.compile(source, name, filename)
File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 565, in compile
self.handle_exception(exc_info, source_hint=source_hint)
File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 754, in handle_exception
reraise(exc_type, exc_value, tb)
File "/usr/lib/python3.4/site-packages/jinja2/_compat.py", line 37, in reraise
raise value.with_traceback(tb)
File "/usr/lib/python3.4/site-packages/fava/templates/context.html", line 16, in template
<dd><code><a href="{{ url_for_source(file_path=context.filename, line=context.lineno) }}" title="{{ _('Show source %(file)s:%(lineno)s', file=context.filename, context.lineno) }}">{{ context.filename }}:{{ context.lineno }}</a></code></dd>
File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 470, in _parse
return Parser(self, source, name, encode_filename(filename)).parse()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 897, in parse
result = nodes.Template(self.subparse(), lineno=1)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 879, in subparse
rv = self.parse_statement()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 121, in parse_statement
return getattr(self, 'parse_' + self.stream.current.value)()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 231, in parse_block
node.body = self.parse_statements(('name:endblock',), drop_needle=True)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 156, in parse_statements
result = self.subparse(end_tokens)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 879, in subparse
rv = self.parse_statement()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 121, in parse_statement
return getattr(self, 'parse_' + self.stream.current.value)()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 203, in parse_if
'name:endif'))
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 156, in parse_statements
result = self.subparse(end_tokens)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 871, in subparse
add_data(self.parse_tuple(with_condexpr=True))
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 616, in parse_tuple
args.append(parse())
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 390, in parse_expression
return self.parse_condexpr()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 395, in parse_condexpr
expr1 = self.parse_or()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 408, in parse_or
left = self.parse_and()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 417, in parse_and
left = self.parse_not()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 428, in parse_not
return self.parse_compare()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 432, in parse_compare
expr = self.parse_add()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 454, in parse_add
left = self.parse_sub()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 464, in parse_sub
left = self.parse_concat()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 474, in parse_concat
args = [self.parse_mul()]
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 484, in parse_mul
left = self.parse_div()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 494, in parse_div
left = self.parse_floordiv()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 504, in parse_floordiv
left = self.parse_mod()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 514, in parse_mod
left = self.parse_pow()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 524, in parse_pow
left = self.parse_unary()
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 543, in parse_unary
node = self.parse_postfix(node)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 672, in parse_postfix
node = self.parse_call(node)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 785, in parse_call
ensure(not kwargs)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 759, in ensure
token.lineno)
File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 50, in fail
raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: invalid syntax for function call expression
The debugger caught an exception in your WSGI application. You can now look at the traceback which led to the error.
To switch between the interactive traceback and the plaintext one, you can click on the "Traceback" headline. From the text traceback you can also create a paste of it. For code execution mouse-over the frame you want to debug and click on the console icon on the right side.

You can execute arbitrary Python code in the stack frames and there are some extra helpers available for introspection:

dump() shows all variables in the frame
dump(obj) dumps all that's known about the object
Brought to you by DON'T PANIC, your friendly Werkzeug powered traceback interpreter.````

**Comment by aumayr** [comment]
Are you running the latest stable version or from `master`? (As you attached a screenshot from pythonanywhere, I assume the latest stable?)

**Comment by redstreet** [comment]
Correct, I'm running the latest stable.

**Comment by yagebu** [comment]
This was fixed in 820070d87290b69754845d347ad54e0b53a49199.

Duplicate of #298
